src/ui/parts/SpriteInfoPart.as

```diff
@@ -75,13 +75,13 @@ public class SpriteInfoPart extends UIPart implements DragClient {
 	}
 
 	public static function strings():Array {
-		return ['direction:', 'rotation style:', 'can drag in player:', 'show:'];
+		return ['direction:', 'rotation style:', 'can drag in fullscreen:', 'show:'];
```

Comment:
When checked, the sprite will be draggable in the player when not in full screen mode.